### DISCUSSION TITLE
Adding ChangeNotifierProvider by list

### DISCUSSION THREAD

**Discussion by ghost** [rrousselGit/provider#719] [Q&A]
I am trying to add a list of ChangeNotifierProviders. I receive no errors but looking at the Inspector is just shows ChangeNotifierProvider\<ChangeNotifier\> instead of the name of the ChangeNotifierProvider\<nameofprovider\> provided in the list. List\<ChangeNotifier\> changeNotifiers = [nameofprovider()]; The Net is constant and shows correctly with ChangeNotifierProvider\<Net\>.

      home: MultiProvider(
          providers: [
            ChangeNotifierProvider(create: (_) => Net()),
            for (ChangeNotifier changeNotifier in changeNotifiers)
              ChangeNotifierProvider(create: (_) => changeNotifier),
          ],
          child: const MainPage(),
        ),

**Comment by tsinis** [comment]
Hey! Can you provide a minimal reproducible example?
